test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -248,6 +249,40 @@ def wrap_obj(obj):
     return wrap_obj
 
 
+def testparams(**kwargs):
+    """
+    Decorator to set or modify test specific parameters
+    """
+    def decorated(function):
+        function.__doc__ += "Test Params:" + "\n\t"
+        for key, value in kwargs.iteritems():
+            function.__doc__ += str(key) + ' : ' + str(value) + '\n\t'
+
+        def wrapper(self, *args):
+            for key, value in kwargs.iteritems():
+                keyname = type(self).__name__ + "." + key
+                if keyname not in self.conf.keys():
+                    self.conf[keyname] = value
+                    self.testconf[keyname] = value
+                else:
+                    self.testconf[keyname] = self.conf[keyname]
+                    t = type(value)
+                    try:
+                        if strtobool(self.conf[keyname]):
```

Comment:
I missed this, it seems strtobool will take `1` or `0` as valid boolean strings, which the test writer could have intended to use them as integers. See https://docs.python.org/2/distutils/apiref.html?highlight=distutils.util#distutils.util.strtobool

So I think you have to do a `if t is bool / else` block.

Reply:
Thanks @vstumpf , Nice catch . Well that means if we use 0 or 1 as integers then also its type will be bool. May be we should make a note of it somewhere not to use 0 or 1 for boolean.

Reply:
Hi @vstumpf, sorry for the delay.  Please review the latest change.